freeipa/src/main/java/com/sequenceiq/freeipa/service/freeipa/user/UserSyncGroupAddMemberOperations.java

```diff
@@ -1,19 +1,21 @@
 package com.sequenceiq.freeipa.service.freeipa.user;
 
+import java.util.Map;
+import java.util.Set;
+import java.util.concurrent.TimeoutException;
+import java.util.function.BiConsumer;
+
+import javax.inject.Inject;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+
 import com.google.common.collect.HashMultimap;
 import com.google.common.collect.Multimap;
 import com.google.common.collect.Multimaps;
 import com.sequenceiq.freeipa.client.FreeIpaClient;
 import com.sequenceiq.freeipa.client.FreeIpaClientException;
-import org.slf4j.Logger;
```

Comment:
why did you rearrange the imports? I think almost 100 files just imports rearrange